src/main/java/ie/gmit/main.java

```diff
@@ -1,5 +1,19 @@
 package ie.gmit;
 
+import javax.swing.*;
+import java.awt.*;
+
 public class main {
 
+    /*
+    public static void main(String[] args) {
+        SwingUtilities.invokeLater(new Runnable() {
+            @Override
+            public void run() {
+                new ResultFrame(false, 100,100,100,100,100);
+            }
+        });
+    }
+
+     */
```

Comment:
why this part is commented out? remove it if its not needed

Reply:
sometimes need